deployments/qat_plugin/overlays/sriov_numvfs/sriov_numvfs_init.yaml

```diff
@@ -7,17 +7,10 @@ spec:
     spec:
       initContainers:
       - name: sriov-numvfs
-        image: busybox
-        command: ["/bin/sh", "-c"]
-        args:
-        - >
-          for dev in `for pf in 0434 0435 19a3 37c8 6f54 4940 18a0; do lspci | awk -v dev="8086:$pf" '$0 ~ dev {print "0000:" $1}'; done`; do
-            DEVPATH="/sys/bus/pci/devices/$dev"
-            NUMVFS="$DEVPATH/sriov_numvfs"
-            if [ -w "$NUMVFS" -a $(cat "$NUMVFS") -eq 0 ]; then
-              cat "$DEVPATH/sriov_totalvfs" | tee "$NUMVFS"
-            fi
-          done
+        image: intel/intel-qat-initcontainer:devel
+        env:
+        - name: ENABLING_PF_PCI_IDs
+          value: "0434 0435 19a3 37c8 6f54 4940 18a0"
```

Comment:
if this default value is added to the script, setting it here is not needed. what do you think?

Reply:
> fair point but it changes the existing functionality without a message anywhere so I thought you just missed my earlier comment

Do you mean this comment?